load_balancer/test/statusTest.js

```diff
@@ -78,7 +78,7 @@ describe('Method getStatus returns correct status object', () => {
 
   it('Correct status object for input array of working nodes', () => {
     nodes = nodesData.Nodes;
-    return checkStatusObject(5, 'up');
+    return checkStatusObject(10, 'up');
```

Comment:
@AnkshitJain This number is equal to the number of execution nodes. It is better to read this number from "$NUMBER_OF_EXECUTION_NODES". This variable is an environment variable and should be available to this node.js process.